src/backend/src/game/game.sse.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Client, SSEService } from '../sse/sse.service';
+import { Response } from 'express';
+import { GameService } from './game.service';
+import { Round } from '../entities/round.entity';
+import { RoundService } from '../round/round.service';
+import { Injectable } from '@nestjs/common';
+
+type GameClientMetadata = { playerId: number; gameId: number };
+const MAX_PLAYERS = 2; // TODO: test temp
+
+@Injectable()
+export class GameSseService {
+  private clients: Record<number, Client<GameClientMetadata>[]> = {};
+  constructor(
+    private gameService: GameService,
+    private roundService: RoundService,
+    private sseService: SSEService<GameClientMetadata>,
+  ) {}
+
+  /**
+   * Subscribe a Client to the Game
+   * @param res
+   * @param metadata
+   */
+  async subscribeClient(
+    res: Response,
+    metadata: GameClientMetadata,
+  ): Promise<void> {
+    this.sseService.subscribeClient(metadata.gameId, { res, metadata });
+  }
+
+  endGame(gameId: number) {
+    // Also send final results
```

Comment:
Is there a reason we haven't added in the relevant final results here?

Reply:
Hmm not sure what else we want to pass? Can we just show a "Game Over" screen?

Reply:
Cool, then lets just get rid of this comment